### DISCUSSION TITLE
OPCUA in Micropython and Raspbeery Pi Pico W

### DISCUSSION THREAD

**Discussion by papercodeIN** [AndreasHeine/opcua-tutorial#2] [Q&A]
Hi, is this possible to use OPC-UA in micropython and Raspberry Pi Pico W Device, and is that possible for you to create something to work on Pi and OPC-UA ?

**Comment by AndreasHeine** [comment]
i dont think so... guess crypto and other libraries are not compatible to the yPython builtins!

**Reply by papercodeIN** [reply]
Okay, Thanks a lot.
